physionet-django/project/views.py

```diff
@@ -1802,8 +1803,8 @@ def published_project(request, project_slug, version, subdir=''):
     an_url = request.get_signed_cookie('anonymousaccess', None, max_age=60 * 60)
     has_passphrase = project.get_anonymous_url() == an_url
 
-    can_view_files = project.can_view_files(user) or has_passphrase
-    is_authorized = project.is_authorized(user) or has_passphrase
+    can_view_project_files = can_view_project_files(project, user) or has_passphrase
```

Comment:
Are you okay with overwriting the function with a boolean? I feel like it would be better to avoid this.

Reply:
sorry, happened with autorefactor, made the change already and force pushed